app/src/app/api/v0/city/[city]/inventory/[year]/route.ts

```diff
@@ -18,9 +20,76 @@ export const GET = apiHandler(async (_req: NextRequest, { params }) => {
   }
   inventory.city = city;
 
-  return NextResponse.json({ data: inventory });
+  let body:Buffer|null = null;
+  let headers:Record<string, string>|null = null;
```

Comment:
You can run prettier (Shift + Option + F on VS Code on Mac to do it for the currently open file, or run `prettier` in the project directory (`app`) to do it for all) to format the code to a common style.